dspace/config/submission-forms.xml

```diff
@@ -59,24 +62,34 @@
                     <repeatable>true</repeatable>
                     <label>Creator</label>
                     <input-type>onebox</input-type>
-                    <hint>Personal name format should be: family, given.</hint>
-                    <required></required>
+                    <hint>The main researchers involved in producing the data in priority order. Personal name format
+                        should be: family, given. As you type a popup will appear showing ORCIDs. If you need to skip
+                        the ORCID selection, hit ESC or click/tap the grey circle icon on the active input field.</hint>
+                    <required>You must fill in the creator(s)</required>
                 </field>
             </row>
-            <!-- Title -->
+            <!-- Title
+             TODO: additional title types?
+             -->
             <row>
                 <field>
                     <dc-schema>dc</dc-schema>
                     <dc-element>title</dc-element>
                     <dc-qualifier></dc-qualifier>
+                    <!-- TODO: more languages in list or remove this completely; map language in DIM2Datacite.xsl
+                    <language value-pairs-name="common_iso_languages">true</language>
+                        -->
```

Comment:
More languages into dc.title?

Reply:
Well the common_iso_languages is just a short list of 10 or so common language. If at some point we want to support  eg. translatedTitles from the full datacite schema, there should be many more languages to pick from